lib/ansible/parsing/mod_args.py

```diff
@@ -178,15 +178,15 @@ def _normalize_parameters(self, thing, action=None, additional_args=dict()):
 
         return (action, final_args)
 
-    def _normalize_old_style_args(self, thing, action):
+    def _normalize_new_style_args(self, thing, action):
         '''
-        deals with fuzziness in old-style (action/local_action) module invocations
+        deals with fuzziness in new style module invocations
+        accepting key=value pairs and dictionaries, and always returning dictionaries
         returns tuple of (module_name, dictionary_args)
 
         possible example inputs:
-            { 'local_action' : 'shell echo hi' }
-            { 'action'       : 'shell echo hi' }
-            { 'local_action' : { 'module' : 'ec2', 'x' : 1, 'y': 2 }}
+            'echo hi', 'shell'
+            {'region': 'xyz'}, 'ec2'
         standardized outputs like:
             ( 'command', { _raw_params: 'echo hi', _uses_shell: True }
```

Comment:
This output comment is incorrect. Also not a new problem, but fix it while you're in the area.